README.md

```diff
@@ -126,21 +126,15 @@ git clone https://github.com/smithoss/Gonymizer.git gonymizer
 ```
 
 4. Build the project
-```
-cd gonymizer/scripts
```

Comment:
i updated this to how i had to build it, I'm not 100% sure if the docs were out of date, or my system is different (im on debian bullseye)